app/usecases/verified.py

```diff
@@ -1,18 +1,33 @@
 from __future__ import annotations
 
-import asyncio
+from datetime import timedelta
+from typing import Optional
 
 import app.state
-import logger
 from config import config
 
-VERIFIED: dict[int, bool] = {}
-FIVE_MINUTES = 60 * 5
+
+async def cache(user_id: int, has_badge: bool) -> None:
+    await app.state.services.redis.set(
+        f"ussr:verified:{user_id}",
+        int(has_badge),
+        timedelta(days=1),
+    )
+
+
+async def get_verified_cache(user_id: int) -> Optional[bool]:
+    cache_badge = await app.state.services.redis.get(
+        f"ussr:verified:{user_id}",
+    )
+
+    if cache_badge:
```

Comment:
unnecessary if check, you can still check if they're equal

Reply:
Fixed